packages/xo-web/src/xo-app/new/network/index.js

```diff
@@ -425,6 +430,16 @@ const NewNetwork = decorate([
                               type='text'
                               value={vlan}
                             />
+                            <label>{_('nbd')}</label>
+                            <Select
+                              name='nbd'
+                              onChange={effects.onChangeNbd}
+                              options={[
+                                { label: _('noNbdConnection'), value: false },
+                                { label: _('nbdConnection'), value: true },
```

Comment:
IMO, to be consistent with this [commit](https://github.com/vatesfr/xen-orchestra/pull/6646/commits/7da0875ba434fa4857b4a54a4dfc0101e51a6630), you should rename `label` here too.

Reply:
No, because here, you're passing the actual label, not just an ID.